api/Jobs.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: drazen.popov
+ * Date: 9/22/2015
+ * Time: 10:28 AM
+ */
+include_once'autoload/boot.php';
+class Jobs {
+
+    public $arg;
```

Comment:
// Please use more descriptive property names. $arg tells nothing about the data it will hold.